app/helpers/layout_helper.rb

```diff
@@ -98,6 +98,28 @@ def javascript(*args)
     content_for(:javascripts) { javascript_include_tag(*args) }
   end
 
+  def javascript_include_tag(*params, **kwargs)
+    # Workaround for overriding javasctipt load with webpack_asset_paths, should be removed when webpack_asset_paths is removed
+    if kwargs.is_a?(Hash) && kwargs[:source] == "webpack_asset_paths"
+      kwargs[:webpacked] 
+    else
+      super(*params, **kwargs)
+    end
+  end
+
+  def webpack_asset_paths(plugin_name, extension: 'js')
+    if extension == 'js'
+      Foreman::Deprecation.deprecation_warning('3.9', '`webpack_asset_paths` is deprecated, use `content_for(:javascripts) { webpacked_plugins_js_for(plugin_name) }` instead.')
```

Comment:
The version passed in is the one where it gets removed. 3.9 implies it should be removed immediately since that's the current nightly.